src/plugins/auto/tests/unit/auto_unit_test.cpp

```diff
@@ -134,6 +134,8 @@ ov::mock_auto_plugin::tests::BaseTest::~BaseTest() {
     mockExeNetworkActual = {};
     config.clear();
     metaDevices.clear();
+    inferReqInternal->Reset_m_compiled_model();
+    inferReqInternalActual->Reset_m_compiled_model();
```

Comment:
we cannot change dev API because of tests.

what is the issue BTW?

Reply:
It is not good enough PR, it will be closed and replaced by another PR: https://github.com/openvinotoolkit/openvino/pull/24372